vertx-web/src/main/java/io/vertx/ext/web/sstore/impl/SessionImpl.java

```diff
@@ -162,7 +219,7 @@ private Buffer writeDataToBuffer() {
       Buffer buffer = Buffer.buffer();
       buffer.appendInt(data.size());
```

Comment:
For the sake of other implementations that reuse the `SessionImpl` class I've also avoid creating a new map while serializing/deserializing.